docs/pvcscale.md

```diff
@@ -0,0 +1,197 @@
+
+### JOB_COMPLETION_POLL_ATTEMPTS
+Default: `360`
+Number of retries for Ansible to poll if the workload job has completed. Poll attempts delay 10s between polls with some additional time taken for each polling action depending on the orchestration host setup.
+
+### PVCSCALE_TEST_PREFIX
+Default: `pvcscale`
+Test to prefix the pbench results.
+
+### PVCSCALE_CLEANUP
+Default: `true`
+Enables/disables cleanup of this workload on completion. Cleanup in
+PVCScale terminology means deletion of `pvcscale0` project - this operation can be
+resource intensive in case there are many PODS/PVCS  in `pvcscale0` project which are created
+during test.
+
+### PVCSCALE_BASENAME
+Default: `pvcscale`
+Basename used by cluster loader for the project(s) it creates.
+
+### PVCSCALE_MAXPODS
+Default: `1`
+Maximum number of pods that will be created by the pvcscale workload. 
+Typically much higher values are used than the default for large scale tests.
+PVCScale does not have implemented mechanism to calculate maximum number of pods which all allowed to be started on 
+particular cluster, so it is necessary to adapt `PVCSCALE_MAXPODS` to cluster capacity and
+execute it accordingly. 
+Choosing `PVCSCALE_MAXPODS` to be high can lead that many PODS/PVCs are created what can affect cluster limits and peformance.
+
+### PVCSCALE_POD_IMAGE
+Default: `quay.io/openshift-scale/scale-ci-workload`
+Pod image to use for pods that are pvscale pods. This image will be used to
+start POD and bound PVC to it.
+
+### PVCSCALE_STEPSIZE
+Default: `1`
+Number of PODS for cluster loader will create before waiting for pods to become running.
+PVCScale test creates one pod/pvc at time.
+
+### PVCSCALE_PAUSE
+Default: `0`
+Period of time (in seconds) for cluster loader to pause after creating pods and 
+waiting for them to be running.PVCScale does not wait on pods to be `Ready` 
+before it creates new ones - it creates pods in fastest possible manner. 
+
+### PVCSCALE_STORAGE_SIZE
+Default: `1Gi`
+This is default PVC size. If specified as `1Gi` then all pods will get `1Gi`
+device mounted inide pods.
+
+### PVCSCALE_STORAGECLASS
+Default: `none`
+It is important to have working storage class prior starting PVCScale test.
+`PVCSCALE_STORAGECLASS` parameter is name of storage class which one plan to
+use, and any storage class supporting dynamic storage provisioning will work
+
+### ACCESS_MODES
+Default: `ReadWriteOnce`
+`ACCESS_MODES` is access_mode for PVC inside container. This field does not
+affect the test itself and might be removed in future. 
+
+
+### Smoke test variables
+
+```
+PVCSCALE_TEST_PREFIX=pvcscale_smoke
+PVCSCALE_CLEANUP=true
+PVCSCALE_BASENAME=pvcscale
+PVCSCALE_MAXPODS=10
+PVCSCALE_POD_IMAGE="quay.io/openshift-scale/scale-ci-workload"
+PVCSCALE_STEPSIZE=1
+PVCSCALE_PAUSE=0
+```
+
+An longer example of variables can be found
+[here](https://gist.githubusercontent.com/ekuric/6038cb7437d58679de00d31b0365ce4c/raw/d7570f0a4499c6f9d9a395ecdd969bf3bc8b59b3/gistfile1.txt)
```

Comment:
Not sure we should use pastes like this, @akrzos ?  Not a fan myself.

Reply:
I think we should just shorten the content in the gist, remove anything which shouldn't be exposed and place inline to readme here.